### PR TITLE
Social Links

### PR DESCRIPTION
**Pull request by frinzekt**
# Share with Socials For Pages and Posts

### Buttons
image

### Seen on Pages
image

### Seen on Posts
image

### Facebook
image

I am not sure if it gives off image... because when I was first reading the documentation, it relied on some SEO image thing which I put like this in the home2health deployed website atm... maybe something with `.now.sh` stuff
image

### Twitter
image

### Linkedin
image

### CODE REVIEW

File: frontend/components/SocialShare.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    : 'https://home2health.org.au/',
+  title = 'Home2Health',
+  text = title,
+}) => {
+  return (
+    <div className="container">
+      <strong>Share With</strong>
+      <a
+        className="btn-floating btn-small waves-effect waves-light light-blue darken-4"
+        target="_blank"
```
Comment (noobling) [inline frontend/components/SocialShare.js:15]: You will want to add ref='noopener noreferrer' https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

File: frontend/pages/medical-recovery-centre/index.js [hunk 2]
```diff
@@ -89,14 +90,15 @@ const style = (
     }
     .card-margin {
       margin: 0 !important;
-    }
+    }import SocialShare from './../../components/SocialShare';
```
Comment (noobling) [inline frontend/pages/medical-recovery-centre/index.js:93]: Should the import be here?

Comment (frinzekt) [reply frontend/pages/medical-recovery-centre/index.js:93]: no it shouldn't be ... i think my auto import is broken xDD

File: frontend/components/SocialShare.js (hunk 1 above)
Comment (frinzekt) [reply frontend/components/SocialShare.js:15]: cool, thanks

Comment (frinzekt) [reply frontend/components/SocialShare.js:15]: @noobling `ref='noopener noreferrer' ` or `rel='noopener noreferrer' `

Comment (noobling) [reply frontend/components/SocialShare.js:15]: It’s rel sorry

**Pull request merged**